deployments/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -23,6 +23,7 @@ spec:
         wcm/nsr.port: {{ .Values.nseControl.nsr.port | quote }}
 {{- end }}
     spec:
+      serviceAccount: {{ .Values.nsm.serviceName }}-acc
```

Comment:
This will be changed to {{ .Values.nsm.serviceName }}-service-account in order to keep consistency with the operator.